new_study_assistant.py

```diff
@@ -0,0 +1,624 @@
+    f.write('<br>\n')
+    f.write('<br>\n')
+
+    f.write(match_table.to_html(index=False))
+
+    f.write('^ represents matches found based on the attribute names<br>')
+    f.write('* represents matches found based on clustering of the attribute values<br>')
+    f.write('NOTE: PATIENT_ID and SAMPLE_ID are omitted as they should be present in every study.')
+    f.write('<br>\n')
+    f.write('<br>\n')
+
+    if predicted_types is not None:
+        f.write('<center><font size=\"5\">Matching attribute types </font></center><br>\n')
+        f.write('Predictions are given as to whether an attribute is a patient or sample attribute.\n')
+        f.write('  The sample/patient prediction is based on what is most common for that particular attribute in the existing cBioPortal studies.\n')
+        f.write('<br>\n')
+        f.write(predicted_types.to_html(index=False)) 
+        f.write('<br>\n')    
+
+    #with doc.create(Subsection('Number of attributes distribution\n')):
+    f.write('<div class="image" style="display:table;">')
+    f.write('<img src=\"' + file_directory + 'n_attribute_distribution.png\"' + 'alt=\"number of attributes\" width=\"800\">')
+    f.write('<br>\n')
+    f.write('<div style="display:table-caption;caption-side:bottom;">Comparison between the number of attributes in the new study and the number of attributes in each existing study on cBioPortal.  The dashed black line indicates the number of attributes in the new study, while the histogram shows the data for existing cBioPortal studies.</div>\n')
+    f.write('</div>\n')
+
+    f.write('<br>')
+
+    f.write('<div class="image" style="display:table;">')
+    f.write('<img src=\"' + file_directory + 'n_unique_attribute_distribution.png\"' + 'alt=\"number of unique attributes\" width=\"800\">')
+    f.write('<br>\n')
+    f.write('<div style="display:table-caption;caption-side:bottom;">Comparison between the number of unique attributes in the new study and the number of unique attributes in each existing study on cBioPortal.  The dashed black line indicates the number of unqiue attributes in the new study, while the histogram shows the data for existing cBioPortal studies.</div>\n')
+    f.write('</div>\n')
+
+    f.write('<br>')
+
+    f.write('<div class="image" style="display:table;">')
+    f.write('<img src=\"' + file_directory + 'n_common_attribute_distribution.png\"' + 'alt=\"number of common attributes\" width=\"800\">')
+    f.write('<br>\n')
+    f.write('<div style="display:table-caption;caption-side:bottom;">Comparison between the number of common attributes in the new study and the number of common attributes in each existing study on cBioPortal.  The dashed black line indicates the number of common attributes in the new study, while the histogram shows the data for existing cBioPortal studies.</div>\n')
+    f.write('</div>\n')
+
+    f.close()
+
+#values that should be read in
+# Parse arguments
+file_directory=''
```

Comment:
We should require the user to provide an output directory to write the results, images, etc. to. You can use `os.path.join(output_directory, <filename_to_write.extension>)` to get the full file path to write to. 

In my comment above,  `os.path.realpath(...)`  will resolve the canonical path, including the root path of the file written out, which is what `webbrowser.open(...)` needs... (just explaining why we have different usages here)